Sources/MongoSwift/MongoNamespace.swift

```diff
@@ -1,11 +1,15 @@
 /// Represents a MongoDB namespace for a database or collection.
-internal struct MongoNamespace {
-    internal let db: String
-    internal let collection: String?
+public struct MongoNamespace: Codable {
+    public let db: String
```

Comment:
this and collection probably need docstrings now that this is a public type. I would just keep them very basic like:
"The database name."
"The collection name if this is a collection's namespace, or nil otherwise`.